### PR TITLE
#170947601 Create the client readme file

### PR DESCRIPTION
**Pull request by Niyonsengaeric**
#### What does this PR do?
 This PR ensures that the README file has all the information required to help a user to use the system.
#### Description of Task to be completed?
- Create a README file to help a user to interact with the system.

#### How should this be manually tested?
- Go to the [README](https://github.com/andela/knights-bn-frontend/tree/ch-client-readme-file-170947601)

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#170947601](https://www.pivotaltracker.com/story/show/170947601)

#### Screenshots
N/A

### CODE REVIEW

**Review by Kagorora — COMMENTED** [review]
@Niyonsengaeric  can you try to give as many details as you can, search as :

1. Tools that you need to use this app.
2. The link of the backend
3. Description of the project
4. Name of the contributors

and other materials that needs to be included

**Review by EugeneMunya — APPROVED** [review]

**Review by Kagorora — APPROVED** [review]

**Comment by T2Wil** [comment]
Hey @Niyonsengaeric , 
consider changing the commit message footer from `start` to `finishes`

**Review by T2Wil — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,7 +1,47 @@
...
+[Ishimwe William](https://github.com/T2Wil)
+
+[Maxime Kagorora](https://github.com/Kagorora)
+
+[Eugene Munyampundu](https://github.com/EugeneMunya)
+
+[Niyonsenga Eric](https://github.com/Niyonsengaeric)
+
+
+ ### TTL
```
Comment (kagabof) [inline README.md]: Please Put TTL in full.

**Review by kagabof — CHANGES_REQUESTED** [review]

**Comment by kagabof** [comment]
Well done.

**Review by kagabof — APPROVED** [review]

**Pull request merged**